cli/requirements.txt

```diff
@@ -1,4 +1,6 @@
 sphinx>=1.3.1, <2.0
 tox>=2.2, <3.0
 wheel>=0.24.0, <1.0
+pytest>=2.9.1
+oauth2client>=1.5.2
```

Comment:
Was needed when we had the original oauth, but we don't need it now! Removed it from install.py also.